spec/integration/metric_spec.rb

```diff
@@ -661,7 +661,7 @@
             end
 
             data = @client.gauges.get_data(hawkular_mem_id)
-            expect(data.size).to be > 2 # needs the services to be running for more than 2 minutes
+            expect(data.size).to be > 2 # needs the services to be running for some time (~15 min)
```

Comment:
I'm not sure I understand this comment:
`# needs the services to be running for some time (~15 min)`
Yet the live tests are passing in under 10 minutes.

Reply:
Interesting... I think collection interval for OS metrics is set to 5 minutes... maybe there's some specific config for travis?

Reply:
Just confirming, it's the purpose of script file in `.travis/build_config_for_testing.rb` So, when cassettes are generated locally without any specific configuration, it's about 15 minutes to wait for 2 datapoints of this metric.